### PR TITLE
New package: GorillaDevs.Ferium version 4.5.0

### LINKED ISSUE #137328: [Package Request]: `GorillaDevs.Ferium`
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

**          | **
------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Download Page Url | https://github.com/gorilla-devs/ferium/releases/tag/v4.5.0
Publisher         | https://github.com/gorilla-devs
Package Name      | `GorillaDevs.Ferium`
Description       | <blockQuote cite="https://github.com/gorilla-devs/ferium/blob/52cd45558e13dc26fc5d3dc9047c3f54a51ea00a/README.md#ferium:~:text=ferium%20is%20a%20fast%20and%20feature%20rich%20cli%20program%20for%20downloading%20and%20updating%20minecraft%20mods%20from%20modrinth%2C%20curseforge%2C%20and%20github%20releases%2C%20and%20modpacks%20from%20modrinth%20and%20curseforge.">Ferium is a fast and feature rich CLI program for downloading and updating Minecraft mods from Modrinth, CurseForge, and GitHub Releases, and modpacks from Modrinth and CurseForge.</blockQuote>
Package Version   | 4.5.0
Installer URL     | https://github.com/gorilla-devs/ferium/releases/download/v4.5.0/ferium-windows-msvc.zip

https://github.com/search?q=repo%3Amicrosoft%2Fwinget-pkgs+ferium&type=commits returned 0 results at +2024-01-31T14:31:24+00:00.

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.0.3 :rocket:

- Resolve: #137328

@stephengillie @ST-DDT @RokeJulianLockhart @theRookieCoder

### CODE REVIEW

**Review by ST-DDT — CHANGES_REQUESTED** [review]

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Review by ST-DDT — APPROVED** [review]

**Review by theRookieCoder — APPROVED** [review]

File: manifests/g/GorillaDevs/Ferium/4.5.0/GorillaDevs.Ferium.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ferium.exe
+UpgradeBehavior: install
+ReleaseDate: 2024-01-30
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/gorilla-devs/ferium/releases/download/v4.5.0/ferium-windows-msvc.zip
+  InstallerSha256: 9649ED1554FC2E03688C132027C99A6611E9A7EF313C3370433B79F4B49E0E8D
```
Comment (Trenly) [inline manifests/g/GorillaDevs/Ferium/4.5.0/GorillaDevs.Ferium.installer.yaml:15]:
```suggestion
  InstallerSha256: 9649ED1554FC2E03688C132027C99A6611E9A7EF313C3370433B79F4B49E0E8D
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
image

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
Does anybody know what the `Validation-Executable-Error` label refers to? It doesn't have a description.

**Comment by Exorcism0666** [comment]
> Does anybody know what the `Validation-Executable-Error` label refers to? It doesn't have a description.
> 
> EDIT:
> 
> > Validation-Executable-Error
> > During installation testing, the test was unable to locate the  primary application. Make sure the application installs correctly on all  platforms. If your application does not install an application, but  should still be included in the repository, add a comment to the pull  request to get the Windows Package Manager engineers to investigate.
> > 
> > 
> > 
> > Source: https://learn.microsoft.com/en-us/windows/package-manager/package/repository
> 
> Though I still don't understand what that means, because the installation sure works for me.

I think the application can't be installed on several architectures like x86, x64, arm64, amd64, etc.... (I may be talking nonsense, but anyway)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 706.)

**Comment by stephengillie** [comment]
Our validation pipeline was really designed with Windows Forms in mind, and gets confused by CLI applications. So it needs a human to verify.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 

(Automated response - build 716.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 5:02:57 PM: 
> 2024-02-08 17:02:56.243 [FAIL] Installer failed security check. Url: https://github.com/gorilla-devs/ferium/releases/download/v4.5.0/ferium-windows-msvc.zip Result: 0x80004005

(Automated response - build 720.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 720.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 729.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-12 15:03:57.062 [FAIL] Installer failed security check. Url: https://github.com/gorilla-devs/ferium/releases/download/v4.5.0/ferium-windows-msvc.zip Result: 0x80004005 MSI (s) (00:9C) [15:03:39:058]: Windows Installer installed the product. Product Name: Microsoft Visual C++ 2022 X64 Minimum Runtime - 14.38.33135. Product Version: 14.38.33135. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (00:D8) [15:03:39:948]: Windows Installer installed the product. Product Name: Microsoft Visual C++ 2022 X64 Additional Runtime - 14.38.33135. Product Version: 14.38.33135. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. [0E9C:178C][2024-02-12T15:03:37]w363: Could not create system restore point, error: 0x80070422. Continuing...

(Automated response - build 729.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 745.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**